crossgovernmentservices/sue-my-brother
# Use HO Keycloak Proxy instead of own OIDC client code

## What
* Demonstrate integration with HO Keycloak Proxy (https://github.com/gambol99/keycloak-proxy)
* Simplify codebase by removing existing OIDC client code

## How to review
1. Browse to https://kc-proxy.cloudapps.digital/ (HO Keycloak Proxy for this branch of SMB)
2. See Sue My Brother front page, click "Sign in"
3. Be taken to our Keycloak instance (https://keycloak.civilservice.digital) to choose an IdP
4. Choose an IdP and login
5. Be taken back to SMB
